tests/ui_tools/test_boxes.py

```diff
@@ -96,22 +96,27 @@ def test_not_calling_typing_method_without_recipients(
         [
             ("#**Stream 1>T", 0, True, "#**Stream 1>Topic 1**"),
             ("#**Stream 1>T", 1, True, "#**Stream 1>This is a topic**"),
-            ("#**Stream 1>T", 2, True, None),
-            ("#**Stream 1>T", -1, True, "#**Stream 1>This is a topic**"),
-            ("#**Stream 1>T", -2, True, "#**Stream 1>Topic 1**"),
-            ("#**Stream 1>T", -3, True, None),
+            ("#**Stream 1>T", 2, True, "#**Stream 1>Hello there!**"),
+            ("#**Stream 1>T", -1, True, "#**Stream 1>He-llo there!**"),
+            ("#**Stream 1>T", -2, True, "#**Stream 1>Hello there!**"),
+            ("#**Stream 1>T", -3, True, "#**Stream 1>This is a topic**"),
```

Comment:
Note that the previous test cases were both forward and backward tests (0 to 2, -1 to -3), which cycles around. This loses the testing of those cases, since the -1 and -2 cases are now presumably the equivalent of the 3 and 4 cases?